Sources/Crypto/Util/Zeroization.swift

```diff
@@ -24,14 +24,14 @@ extension UnsafeMutablePointer: Zeroization {
     /// Zeroizes the pointee
     func zeroize() {
         let size = MemoryLayout.size(ofValue: Pointee.self)
-        memset_s(self, size, 0, size)
+        _ = memset_s(self, size, 0, size)
     }
 }
 
 extension Array: Zeroization where Element == UInt8 {
     /// Zeroizes the array
     mutating func zeroize() {
-        memset_s(&self, self.count, 0, self.count)
+        _ = memset_s(&self, self.count, 0, self.count)
```

Comment:
Can we revert these two changes and mark our `memset_s` as `@discardableResult`? That's implicitly applied to C functions elsewhere.

Reply:
Thank you for the review. I'll fix it.